api/spec/src/billing/invoices.tsp

```diff
@@ -398,7 +404,17 @@ enum InvoiceExpand {
 @friendlyName("BillingInvoiceCreateInput")
 model InvoiceCreateInput {
   /**
-   * The pending items to include in the invoice, if not provided all pending items will be included.
+   * The pending line items to include in the invoice, if not provided:
+   * - all line items that have invoice_at < now() will be included
+   * - all usage based line items will be included up to now(), new usage-based line items will be staged for the rest
+   *   of the billing cycle
+   */
+  IncludePendingLines?: ULID[];
+
+  /**
+   * The time as of which the invoice is created.
+   *
+   * If not provided, the current time is used.
    */
-  IncludePendingItems?: ULID[];
+  AsOf?: DateTime;
```

Comment:
Defaults to now.

Reply:
Only scalar types can be defaulted, DateTime is not a scalar type.